base/uk.ac.stfc.isis.ibex.ui.graphing/src/uk/ac/stfc/isis/ibex/ui/graphing/websocketview/MatplotlibFigure.java

```diff
@@ -114,8 +134,20 @@ public void mouseMove(MouseEvent e) {
 			}
 		};
 		
+		mouseListener = new MouseAdapter() {@Override
+			public void mouseDown(MouseEvent e) {
+				viewModel.notifyButtonPressed("button_press");
+			}
+			
+			@Override
+			public void mouseUp(MouseEvent e) {
+				viewModel.notifyButtonPressed("button_release");
```

Comment:
I think we should make `button_press` and `button_release` an enum so that we can later add things like double_click easily if we need to